### PR TITLE
Add wiki.gg icon

### PR DESCRIPTION
**Pull request by alistair3149**
wikidotgg

**Similarweb rank:**
https://www.similarweb.com/website/wiki.gg - `#3,878`

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
* SVG taken from official source: https://commons.wiki.gg/wiki/Category:Wiki.gg_logos
* Brand color from the official pink logo: `#FF1985`

### CODE REVIEW

**Review by LitoMore — CHANGES_REQUESTED** [review]
image

Hi @alistair3149, I found some differences. I used https://commons.wiki.gg/wiki/File:Wiki.gg_logo-pink.svg as a reference.

Which one did you use as reference?

**Comment by alistair3149** [comment]
I used the [black one](https://commons.wiki.gg/wiki/File:Wiki.gg_logo-black.svg).

**Comment by LitoMore** [comment]
@alistair3149 Could you use the pink one? Since you're submitting the pink color as the brand color.

**Comment by alistair3149** [comment]
Done

**Review by pisceskaze — APPROVED** [review]

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13438,6 +13438,11 @@
             "hex": "8B8B8B",
             "source": "https://commons.wikipedia.org/wiki/File:WiiU.svg"
         },
+        {
+            "title": "Wiki.gg",
```
Comment (LitoMore) [inline _data/simple-icons.json:13442]:
```suggestion
            "title": "wiki.gg",
```

Their brand name is in lowercase.

Comment (LitoMore) [reply _data/simple-icons.json:13442]: image

@alistair3149 The shape is a perfect fit. But the icon name should be corrected.

**Review by LitoMore — APPROVED** [review]

**Pull request merged**